src/appleseed.studio/mainwindow/pythonconsole/pythoneditor.h

```diff
@@ -47,15 +47,11 @@ class PythonEditor
 {
     Q_OBJECT
 
+  friend LineNumberArea;
```

Comment:
Didn't know this was legal :) It appears to be a C++11 extension. In C++03 it would have been `friend class LineNumberArea`.